pinot-controller/src/main/java/org/apache/pinot/controller/api/resources/PinotTableRestletResource.java

```diff
@@ -214,8 +214,8 @@ public String alterTableStateOrListTableConfig(
         ObjectNode offline = JsonUtils.newObjectNode();
         tableExists = true;
 
-        offline.put(FileUploadPathProvider.TABLE_NAME, offlineTableName);
-        offline.set(FileUploadPathProvider.STATE,
+        offline.put("tableName", offlineTableName);
```

Comment:
These Strings can be static variables in this class.

Reply:
I considered that, but these 2 constants only apply to this API, and should not be changed for backward-compatibility, thus prefer keeping them like this.